src/Lithnet.GoogleApps/GmailServiceRequestFactory.cs

```diff
@@ -21,9 +23,13 @@ public class GmailServiceRequestFactory
 
         private readonly string[] scopes;
 
-        private readonly OrderedDictionary cache = new OrderedDictionary(100);
+        private const int CacheSize = 10;
```

Comment:
This was 100 to try can keep a decent number of clients for users in memory, as parallel exports can make multiple calls per user. Did you see any problems with this number? I want to try to ensure that a client hangs around for at least the lifetime of it's export item thread.

Reply:
Yes, I suspected that you wanted to keep them in cache mem. I saw problems with Backend errors and also hanging threads (due to InfiniteTimeout). But now InfiniteTimeout is not used, and this should be fine (I just missed changing it back).

Changing back to 100.